server/test/unit/server/webservices/views/test_decorators.py

```diff
@@ -1,14 +1,14 @@
 """
-This module contains tests for the pulp.server.webservices.controllers.decorators module.
+This module contains tests for the pulp.server.webservices.views.decorators module.
 """
 import mock
 
 from .... import base
 from pulp.server.exceptions import PulpCodedAuthenticationException
-from pulp.server.webservices.controllers import decorators
+from pulp.server.webservices.views import decorators
 
 
-class TestAuthenticationMethods(base.PulpWebserviceTests):
+class TestAuthenticationMethods(base.PulpServerTests):
```

Comment:
I agree the class name should be the same, but why did the parent class change? It was `base.PulpWebserviceTests` but not it is `base.PulpServerTests`.